packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -224,6 +226,12 @@ export class FilamentViewer extends LitElement {
 
     this[$updateSize]();
 
+    /*
+    this[$renderer].setClearOptions(
+        {clearColor: [r, g, b, 1], clear: true, discard: true});
+    */
+    this[$view].setClearColor([1000, 1000, 1000, 1]);
```

Comment:
Set it higher because of tone mapping

Reply:
This is a good comment to include in the code.

Reply:
As follow up to our discussion today. note that all Filament actions must take place after the init's callback function has been triggered, as per https://google.github.io/filament/webgl/reference.html#init.

Reply:
Yes, and also let the record show that this was my bug from when I first integrated filament.js; apparently I was just lucky enough to never trigger a race.